crates/polars-lazy/src/frame/mod.rs

```diff
@@ -656,15 +656,44 @@ impl LazyFrame {
         Ok(())
     }
 
+    /// Stream a query result into a parquet file on an ObjectStore-compatible cloud service. This is useful if the final result doesn't fit
+    /// into memory, and where you do not want to write to a local file but to a location in the cloud.
+    /// This method will return an error if the query cannot be completely done in a
+    /// streaming fashion.
+    #[cfg(all(feature = "cloud", feature = "parquet"))]
```

Comment:
I want this feature split in `cloud_write`, as I want to activate reading in python polars, but not writing sinks.